File: TransCoder_tokenized_test_set_functions/python/SORT_GIVEN_STRING_USING_CHARACTER_SEARCH.py
def sortString ( str, n ) :
    new_str = ""
    for i in range ( ord ( 'a' ), ord ( 'z' ) + 1 ) :
        for j in range ( n ) :
            if ( str [ j ] == chr ( i ) ) :
                new_str += str [ j ]
    return new_str

File: TransCoder_tokenized_test_set_functions/java/SORT_GIVEN_STRING_USING_CHARACTER_SEARCH.java
static String sortString ( String str, int n ) {
  String new_str = "";
  for ( int i = 'a';
  i <= 'z';
  i ++ ) for ( int j = 0;
  j < n;
  j ++ ) if ( str . charAt ( j ) == i ) new_str += str . charAt ( j );
  return new_str;
}